packages/core/src/external/opensearch/semantic/shared.ts

```diff
@@ -0,0 +1,23 @@
+import { SearchSetBundle } from "@metriport/shared/medical";
+import { ConsolidatedSnapshotRequestSync } from "../../../command/consolidated/get-snapshot";
+import { buildConsolidatedSnapshotConnector } from "../../../command/consolidated/get-snapshot-factory";
+import { getConsolidatedSnapshotFromS3 } from "../../../command/consolidated/snapshot-on-s3";
+import { Patient } from "../../../domain/patient";
+
+export async function getConsolidated({ patient }: { patient: Patient }): Promise<SearchSetBundle> {
+  // TODO eng-41 always without any filters?
```

Comment:
Does this mean you will add these back in a follow PR before going to prod?

Reply:
> Does this mean you will add these back in a follow PR before going to prod?

This is a new function, it's born without these parameters - as the semantic search is. We're not planning to add those, right?